validator/README.md

```diff
@@ -61,7 +61,7 @@ Install these packages using apt-get:
 - `npm`
 - `openjdk-7-jre`
 - `protobuf-compiler`
-- `python2.7`
+- `python3`
 
 Then use pip to `pip install protobuf`.
```

Comment:
IIUC, using the python3 version of pip might require explicitly invoking `pip3` instead of `pip`.

Reply:
Done, thanks. Also added another package per https://www.educative.io/edpresso/installing-pip3-in-ubuntu. :shrug: